docs/cvl/methods.md

```diff
@@ -722,6 +728,26 @@ Before this version, this may cause vacuous results.
 ```{note}
 `DISPATCHER` summaries cannot be used to summarize library calls.
 ```
+(dispatch-list)=
+#### `DISPATCH` list summaries
+Similar to `DISPATCHER` which dispatches an unresolved method call to all contracts
+implementing the relevant function, we can also dynamically dispatch unresolved calls
+to a user-specified list of possible implementations.
+
+This is most commonly useful on calls where the method sighash is unresolved, see 
+{ref}`catch-unresolved-calls-entry` for a detailed example in such a case.
+
+We can also use the `DISPATCH` list on {ref}`wildcard-methods-entries`, to
+restrict which contract's implementations of the method to consider:
+```cvl
+methods {
+    function _.foo() external => DISPATCH(true)[ C._, D._ ];
+}
+```
+This will dispatch unresolved calls of a method `foo` to implementations in contracts
+`C` and `D`. 
```

Comment:
```suggestion
This will dispatch unresolved calls of a method `foo` to implementations in contracts
`C` and `D`.  Note, if `C` and `D` are the only contract implementing `foo` in the scene, then this is equivalent to using `DISPATCHER(true)`. 
```